Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h

```diff
@@ -86,6 +86,7 @@ enum class SDKAlignedBehavior {
     RestrictsBaseURLSchemes,
     RunningBoardThrottling,
     ScrollViewContentInsetsAreNotObscuringInsets,
+    SearchInputIncrementalAttributeAndSearchEventDisabled,
```

Comment:
Maybe we should call this `NoSearchInputIncrementalAttributeAndSearchEvent` for consistency with the other removed API behaviors?